config/database.pgsql

```diff
@@ -6,7 +6,8 @@ CREATE TABLE IF NOT EXISTS guild_settings(
     rainbow_line_autodelete BOOLEAN DEFAULT FALSE,
     leaderboard_message_url VARCHAR(150),
     dump_stackoverflow_answers BOOLEAN DEFAULT FALSE,
-    nickname_banned_role_id BIGINT
+    nickname_banned_role_id BIGINT,
+    quote_reactions_needed VARCHAR(50) DEFAULT 3
```

Comment:
If you want this to be an integer then make it an integer - `INTEGER` (or `SMALLINT` would be better for this case) for ints rather than `VARCHAR` which is for strings.